cdap-proto/src/main/java/io/cdap/cdap/proto/SourceControlMetadataDetail.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto;
+
+/**
+ * This class is used for responses of API calls that fetch source control metadata.
+ */
+public class SourceControlMetadataDetail {
```

Comment:
Yes, we could but `SourceControlMetadataRecord` has a lot of extra fields. Hence, I had created `SourceControlMetadataDetail`. I'll make the changes since it is fine to return the record

Reply:
done